packages/cf-component-card/src/CardBlock.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  padding: '1rem 1.4rem 1.26667rem',
+  borderTop: `1px solid ${theme.colorGrayLight}`,
```

Comment:
Ah! Now I remember something.

As far as I can remember I couldn't find any instances of where there are two `CardControl` components next to each other so I added it in the example. Since I'm not sure how it is supposed to look I took a stab at doing this by doing the above changes.

It seems to be working like expected even tho it's a child selector and not a sibling selector:

![image](https://user-images.githubusercontent.com/318208/28110333-f4689f60-66e9-11e7-8044-92ee4b89c23a.png)

I feel like my understanding of `:first-child` is that it's not about children of the card block but the first child card block in a parent.

[According to this jsbin they are equivalent](http://jsbin.com/joxixosobo/edit?html,css,output)

With first child: 
![image](https://user-images.githubusercontent.com/318208/28110016-eaca85be-66e8-11e7-9fbe-8a48866f9629.png)

Without first child:
![image](https://user-images.githubusercontent.com/318208/28109998-de45c754-66e8-11e7-9246-55aa1aab64aa.png)